packages/design-system-react/CHANGELOG.md

```diff
@@ -23,6 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fixed Avatar components shrinking in flex layouts ([#825](https://github.com/MetaMask/metamask-design-system/pull/825))
```

Comment:
This PR was included in the `0.4.0` release when `main` was merged into the release branch 

<img width="947" height="490" alt="Screenshot 2025-09-17 at 8 44 40 AM" src="https://github.com/user-attachments/assets/b0771bd0-e938-48c9-9c80-d9895190ec58" />